src/app/[locale]/(main)/quizbank/[id]/components/recommend-quizbank.tsx

```diff
@@ -0,0 +1,164 @@
+          <div className="flex flex-col items-start py-6 pl-6 pr-20 max-md:px-5">
+            <div className="whitespace-nowrap text-base font-semibold leading-6 text-zinc-950">
+              Lorem isplum
+            </div>
+            <div className="mt-1.5 text-sm leading-5 text-zinc-500">
+              320 cards
+            </div>
+          </div>
+        </div>
+        <div className="flex flex-1 flex-col justify-center rounded-3xl border border-solid border-[color:var(--color-border,#E4E4E7)] bg-white shadow">
+          <div className="flex flex-col items-start py-6 pl-6 pr-20 max-md:px-5">
+            <div className="whitespace-nowrap text-base font-semibold leading-6 text-zinc-950">
+              Lorem isplum
+            </div>
+            <div className="mt-1.5 text-sm leading-5 text-zinc-500">
+              320 cards
+            </div>
+          </div>
+        </div>
+        <div className="flex flex-1 flex-col justify-center rounded-3xl border border-solid border-[color:var(--color-border,#E4E4E7)] bg-white shadow">
+          <div className="flex flex-col items-start py-6 pl-6 pr-20 max-md:px-5">
+            <div className="whitespace-nowrap text-base font-semibold leading-6 text-zinc-950">
+              Lorem isplum
+            </div>
+            <div className="mt-1.5 text-sm leading-5 text-zinc-500">
+              320 cards
+            </div>
+          </div>
+        </div>
+        <div className="z-10 flex flex-1 flex-col justify-center rounded-3xl border border-solid border-[color:var(--color-border,#E4E4E7)] bg-white shadow">
+          <div className="flex flex-col items-start py-6 pl-6 pr-20 max-md:px-5">
+            <div className="whitespace-nowrap text-base font-semibold leading-6 text-zinc-950">
+              Lorem isplum
+            </div>
+            <div className="mt-1.5 text-sm leading-5 text-zinc-500">
+              320 cards
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* author */}
+      <div className="mt-16 w-[849px] border-b-2 border-gray-300 text-xl font-bold leading-8 text-black max-md:mt-10 max-md:max-w-full" />
+      <div className="mt-16 text-xl font-bold leading-8 text-black max-md:mt-10 max-md:max-w-full">
+        {i18n("belong_to")}
+      </div>
+      <div className="ml-3.5 mt-6 flex w-[874px] max-w-full justify-between gap-5 px-5 font-medium max-md:flex-wrap">
+        <div className="flex-auto text-2xl leading-10 text-black">
+          {i18n("author.class_title")}:{" "}
+          <span className="font-bold">Eva’s class Biology</span>
+        </div>
+      </div>
+
+      <div className="ml-3.5 flex w-[874px] max-w-full justify-between gap-5 pr-6 max-md:flex-wrap max-md:pr-5">
+        <div className="flex justify-between gap-2 whitespace-nowrap">
+          <img
+            loading="lazy"
+            srcSet="..."
+            className="my-auto aspect-square w-8"
+          />
```

Comment:
Replace with `Avatar` component

Reply:
backend is still error in return back Author value (null), so that is a placeholder
![image](https://github.com/blackmouse572/quizler-fe/assets/87664605/a9b7de0f-9d9c-4f91-abcc-706b15f4b742)